### PR TITLE
added solution to hw

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <properties>
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
+        <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
```
Comment (Alexmansar) [inline pom.xml]: why we need it?

File: pom.xml [hunk 2]
```diff
@@ -20,6 +21,17 @@
...
         </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.7.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.30</version>
```
Comment (Alexmansar) [inline pom.xml]: you can add this as a constant with this vervion, and reuse it

File: src/main/java/dao/UserDataBaseDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create connection to DB", e);
+        }
+        return null;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteTicketsSql = "DELETE FROM ticket_data_base WHERE user_id = ?";
+        String deleteUserSql = "DELETE FROM user_data_base WHERE id = ?";
```
Comment (Alexmansar) [inline src/main/java/dao/UserDataBaseDaoImpl.java]: in general good practice add it as constants

File: pom.xml [hunk 4]
```diff
@@ -63,6 +75,20 @@
...
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
+                            <version>1.18.30</version> <!-- Версия Lombok -->
```
Comment (Alexmansar) [inline pom.xml]: unnecessary comments

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak Hi, fix small comments

File: pom.xml (hunk 1 above)
Comment (AnyaRatusnyak) [reply pom.xml]: Ok , sorry) Thank you!

**Pull request merged**